packages/feedback/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "feedback",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~8.2.14",
+    "@angular/common": "~8.2.14",
+    "@angular/compiler": "~8.2.14",
+    "@angular/core": "~8.2.14",
+    "@angular/forms": "~8.2.14",
+    "@angular/platform-browser": "~8.2.14",
+    "@angular/platform-browser-dynamic": "~8.2.14",
+    "@angular/router": "~8.2.14",
+    "angular-truncate-pipes": "0.0.7",
+    "apollo-angular": "^1.8.0",
+    "apollo-angular-link-http": "^1.9.0",
+    "apollo-cache-inmemory": "^1.6.5",
+    "apollo-client": "^2.6.8",
+    "apollo-link": "^1.2.11",
+    "apollo-link-error": "^1.1.12",
+    "apollo-link-retry": "^2.2.15",
+    "apollo-link-ws": "^1.0.19",
+    "bootstrap": "^4.4.1",
+    "font-awesome": "^4.7.0",
+    "graphql": "^14.5.0",
+    "graphql-tag": "^2.10.3",
+    "ng-pipes": "^0.1.5",
```

Comment:
`ng-pipes` is not the correct package its `ngx-pipes` package. i see that its already installed so you can remove this module

Reply:
Removed.